network/debinterfaces/activate.go

```diff
@@ -110,7 +110,7 @@ fi
 // for the new bridges.
 func BridgeAndActivate(params ActivationParams) (*ActivationResult, error) {
 	if len(params.Devices) == 0 {
-		return nil, errors.Errorf("no devices specified")
+		return nil, errors.New("no devices specified")
```

Comment:
We should probably use the juju error package in this non-test file.

Reply:
Done & Done